randovania/data/json_data/prime1.txt

```diff
@@ -1,6 +1,18 @@
 ====================
 Templates
 
+* Can Boost:
+      Morph Ball and Boost Ball
+
+* Can use Spider Ball:
+      Morph Ball and Spider Ball
+
+* Can lay bomb:
+      Morph Ball Bomb and Morph Ball
+
+* Can lay power bomb:
+      Power Bomb and Morph Ball
+
```

Comment:
These templates might not be necessary and just be extra clutter. See Life Grove for example. Boost, Bombs, and PBs are needed, and thus morph ball is checked for three times.

Reply:
what's about adding can use spinner connector?

Reply:
As a template or a node?

Reply:
rather as a template

Reply:
For now, it's best to list all the requirements. Templates can come later.